llvm/lib/ProfileData/InstrProfReader.cpp

```diff
@@ -38,6 +38,9 @@
 
 using namespace llvm;
 
+// Maxium counter value 2^56.
+static uint64_t MaxCounterValue = 0xffffffffffffff;
```

Comment:
Constant should be declared in a header

Reply:
Something like `(1ULL << 56)` would be easier to read than trying to count the number of `f`'s (which are technically 2^56-1 ;)